docs/pages/tools/sdk-react/useCreateNFT.md

```diff
@@ -36,7 +36,6 @@ export default function Component() {
             content: azukiImage, // content file uploaded by the user
             preview: azukiImagePreview, // preview in the case of private or animation content uploaded by user
```

Comment:
Preview should also be removed
```suggestion
            preview: azukiImagePreview, // preview in the case of animation content
```

Reply:
preview is still useful if we set an animation

Reply:
Yeah I notice this after posting this first suggestion but forgot to change the message!